### DISCUSSION TITLE
How do I set environment variables on [Aug 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3165] [Discord Threads]
> 🙋‍♂️ **habochur**

**Comment by o1lab** [comment]
> 🙋‍♂️ **CaptainKranch**  
#### you can sear it on the search bar  
![image0](https://cdn.discordapp.com/attachments/1005719137986822234/1005970737963728926/unknown.png)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1005719137986822234)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
